readme.md

```diff
@@ -65,6 +65,8 @@ By convention, when the artifact is a library, the files are called
   technique
 - `low-coverage` Script that removes all the sparse fields that have
   coverage less than a given threshold coverage.
+- `stacked-predictions` Script that builds several predictors and
+  returns an appended dataset with the most popular prediction
```

Comment:
just a nit, but was is "an appended dataset"? i'd say "a copy of the original dataset with the most popular prediction appended/added" (or somesuch)